tests-functional/tests/test_settings.py

```diff
@@ -0,0 +1,508 @@
+import logging
+import re
+import time
+
+import pytest
+from clients.api import ApiResponseError
+import datetime
+
+# import time
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.rpc
+class TestSettings:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.config = backend_new_profile("sender")
+
+    def test_verify_node_config_stability(self):
+        cfg1 = self.config.settings_service.get_node_config()
+        cfg2 = self.config.settings_service.get_node_config()
+        assert cfg1 == cfg2, "NodeConfig should be stable across calls"
+
+    def test_check_node_config_params(self):
+        cfg = self.config.settings_service.get_node_config()
+        boot_api = self.config.get_boot_api_config()
+        assert cfg["WSEnabled"] == boot_api["wsEnabled"]
+        assert cfg["WSHost"] == boot_api["wsHost"]
+        assert cfg["WSPort"] == boot_api["wsPort"]
+        assert cfg["HTTPEnabled"] == boot_api["httpEnabled"]
+        assert cfg["HTTPPort"] == boot_api["httpPort"]
+
+    def test_verify_node_config_enforce(self, backend_new_profile):
+        forced_network_id = 4242
+        forced = backend_new_profile("forced_network", network_id=forced_network_id)
```

Comment:
`forced` sound like a flag or smth
```suggestion
        backend = backend_new_profile("forced_network", network_id=forced_network_id)
```

Reply:
Fixed at https://github.com/status-im/status-go/pull/6984/commits/fab70cd39534aae456314e78932b456f43ba34ab